### DISCUSSION TITLE
User group

### DISCUSSION THREAD

**Discussion by Manfredzimmermann** [open-webui/open-webui#3656] [General]
Open WebUI has user management. You can create many users individually or via a csv file. Can a user from a department upload many documents for all other employees in the same department (RAG-Chromadb) so that the others have access to these files or does an employee have to upload the documents, then export the JSON file and the other employees in the same department import this JSON file? As an administrator, can you create a user group for each department and add the employees in the departments? Why I ask this: A user group makes life a lot easier. Let's say we have a marketing department with 20 employees. An employee is given the task of uploading all the documents from the marketing department so that everyone can access this data. What would be really cool is if the employee uploads all the documents and then selects the marketing user group so that the documents that have just been uploaded are automatically available to the other employees. This means that one employee uploads all the documents and the other users in the same department have immediate access to these files. It's a pity that you can't connect Open WebUI to an Active Directory (LDAP)!

**Comment by tdiprima** [comment]
> It's a pity that you can't connect Open WebUI to an Active Directory (LDAP)!

You can.  See this page: [environment variables for ldap](https://docs.openwebui.com/getting-started/env-configuration#ldap).

If you're struggling with Active Directory, check the GitHub issue [LDAP Not Working With Active Directory · Issue #7063](https://github.com/open-webui/open-webui/issues/7063) for troubleshooting tips. It mentions things like double-checking AD-specific configs like the username attribute (`sAMAccountName`) and ensuring the LDAP server URL (e.g., `ldap://` or `ldaps://`) is correct.

Although, there's no mention of user groups, document uploads, RAG, Chromadb, or sharing docs within departments. It's all about getting LDAP login to work, not user management or file-sharing features.

**Comment by tdiprima** [comment]
So how about it... anybody got any ideas? 😎

Can a department employee upload docs to RAG/Chromadb and auto-share them with their department? Can admins create department user groups to make this easier, like one person uploads docs and picks the "Marketing" group to instantly share with all 20 team members?

**Reply by Classic298** [reply]
yes this is possible. Upload it to a knowledge collection, and that knowledge collection is shared with a group.

**Reply by flefevre** [reply]
I do confirm

- Create a knowledge base
- Share it with your group with read or write rights
- Upload files in order to embedded and vectorise them
- Access the knowledge base through # in the prompt

My advice will be to create a model assistant
